google/cloud/security/common/data_access/sql_queries/select_data.py

```diff
@@ -65,10 +65,16 @@
     SELECT group_id from groups_{0};
 """
 
-GROUP_USERS = """
+GROUP_ID = """
+    SELECT group_id
+    FROM groups_{1}
+    WHERE group_email = "{0}";
+"""
+
+GROUP_MEMBERS = """
     SELECT group_id, member_role, member_type, member_id, member_email
-    FROM group_members_{0}
-    WHERE member_status = "ACTIVE" and member_type = "USER";
+    FROM group_members_{1}
+    WHERE group_id = "{0}";
```

Comment:
Correct, you would still use the .format() for the timestamp. However, format does not protect against sql injection, so you'd want to do something like this:

```
sql = select_data.GROUP_MEMBERS.format(timestamp)
cursor.execute(sql, group_id)
```

where GROUP_MEMBERS is:

```
GROUP_MEMBERS = """
      SELECT group_id, member_role, member_type, member_id, member_email		      
      FROM group_members_{}
      WHERE group_id = %s;
```

(more information: http://bobby-tables.com/python, http://stackoverflow.com/questions/7929364/python-best-practice-and-securest-to-connect-to-mysql-and-execute-queries)